community/kernel/src/main/java/org/neo4j/kernel/impl/store/record/Record.java

```diff
@@ -45,16 +45,17 @@
     NO_LABELS_FIELD( (byte)0, 0 );
 
     public static final byte CREATED_IN_TX = 2;
+    public static final byte REQUIRE_SECONDARY_UNIT = 4;
```

Comment:
We need to store two bits, one is whether or not secondary unit is _required_, the other one is is whether or not the record _has been assigned_ a secondary unit. The difference between the two is covered in comments in `AbstractBaseRecord` requiring a secondary unit implies having a secondary unit, but it's not the other way around.